docs/tutorials/runtime-module-in-own-crate.md

```diff
@@ -0,0 +1,133 @@
+```bash
+cargo test
+```
+
+Turns out we'll need some additional dependencies to make the test pass. You can see this from the error messages, or by looking at the test module itself. Let's add `primitives`, `runtime-primitives`, and `runtime-io` as dev dependencies. The dev dependencies do not need to be listed as part of `std`. TODO is that even proper phrasing?
+
+Add these lines to your `Cargo.toml`.
+```toml
+
+[dev-dependencies.primitives]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'substrate-primitives'
+rev = 'bb68456966bf9d767651773dbaadd6787bce1884'
+
+[dev-dependencies.runtime-primitives]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'sr-primitives'
+rev = 'bb68456966bf9d767651773dbaadd6787bce1884'
+
+[dev-dependencies.runtime-io]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'sr-io'
+rev = 'bb68456966bf9d767651773dbaadd6787bce1884'
+```
+
+We're now good to go for this particular module. It will take a little experimentation to gain familiarity with what dependencies you need for each module you dream up. So the take-away from this section is not that `Cargo.toml` should always look like it did today. Rather, `Cargo.toml` needs to have the correct dependencies and you now know a few places to look for how to specify them when you get stuck.
+
+## Add your Runtime Module to your Node
+With our runtime module now compiling and passing tests (well, passing test anyway) we're ready to add it to our node template.
+
+First we need to add our newly-created crate as a dependency in the node runtime's `Cargo.toml`.
+
+```toml
+[dependencies.test_module]
+default_features = false
+path = "../../test_module"
+# Optionally specify a version
+# version = "0.1.0"
+```
+
+And remember to update the `std` section/
+```toml
+'test_module/std'
+```
+
+Next we'll update `runtime/lib.rs` in modular-chain to actually use the new runtime module. The lines that bring in the template module from when it was coded directly in the node runtime are still there, so we just need to update them in three places.
+
+1. Remove `mod template` as we've already brought that dependency in via `Cargo.toml`
+2. Update the trait implementation
+```rust
+/// Used for the module test_module
+impl test_module::Trait for Runtime {
+	type Event = Event;
+}
+```
+
+3. Update `construct_runtime`
+```rust
+TestModule: test_module::{Module, Call, Storage, Event<T>},
+```
+
+## Appreciate your Work
+At this point you have the template module packaged up as it's own crate, and running as part of your `modular-node`. You can now run your chain with `./target/release/modular-chain --dev` and start the [Apps UI](https://polkadot.js.org/apps/#/explorer) to confirm that the module is working as expected. If you aren't familiar with the Apps UI, you will need to navigate to the settings tab, and select 'Local Node'.
```

Comment:
You should probably mention the user needs to run `./scripts/build.sh` and `cargo build --release` and `purge-chain`